ClientHandler.py

```diff
@@ -11,8 +11,16 @@ def Close(self):
         print(self, 'Client disconnected')
 
     def Network_message(self, data):
-        print(self, data)
+        print(self, "Network_message work!")
+
+    def PlayerInfo(self, playerData):
+        # print(playerData.id)
+        print("Send player info!")
+        self.Send({"action": "playerInfo", "playerInfo": dict({"id": playerData.id, "x": playerData.x, "y": playerData.y})})
+
+    def PlayersDict(self, data):
+        self.Send({"action": "playersDict", "playersDict": data})
 
     def Message(self, data):
-        print(self, "Sending message")
+        # print(self, "Sent all players info")
         self.Send({"action": "message", "message": data})
```

Comment:
delete these commented prints if we dont need them

Reply:
Deleted